### ISSUE TITLE
TreeView: Drag and drop via keyboard does not work with JAWS

### ISSUE THREAD

**Issue by matuzalemsteles** [liferay/clay#5385]
Evan did a test using JAWS and moving the drop target using the arrow keys does not work because JAWS may be suppressing keyboard events.

**Comment by matuzalemsteles** [comment]
After some investigation I noticed that this is related to JAWS suppression of keyboard events according to accessibility rules for tree, for example rendering indicators using ARIA `tree` does not work properly listening for keydown events and moving between elements.

In order to be able to dnd correctly to JAWS we need to switch to the `treegrid` accessibility pattern https://www.w3.org/WAI/ARIA/apg/patterns/treegrid/examples/treegrid-1/. I did a small test for the first level by changing only the ARIA attributes and they work much more properly but as said we need to virtualize the data to a list to be able to implement the treegrid accessibility pattern which is a list instead of a tree.

To fix this problem we need to work on issue #5030 first. I'm removing this issue from the release due to the block and raising the priority from #5030.

**Comment by liferay-peds** [comment]
LPS Ticket: https://liferay.atlassian.net/browse/LPS-190181
